### PR TITLE
Add a verbosity guide

### PR DESCRIPTION
**Pull request by Gedochao**
Context: https://github.com/VirtusLab/scala-cli/pull/1920#discussion_r1139381785
~Depends on #1920~

### CODE REVIEW

File: website/docs/guides/verbosity.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+import {ChainedSnippets} from "../../src/components/MarkdownComponents.js";
+
+Logging in Scala CLI can be controlled in a number of ways.
+
+```scala title=Hello.sc
+println("Hello")
+```
+
+Some things will be printed by default.
```
Comment (lwronski) [inline website/docs/guides/verbosity.md]: I think, we can add what things we print, maybe something like this?

```suggestion
Logs, warning and error will be printed by default.
```

File: website/docs/guides/verbosity.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+```text
+Hello
+```
+
+</ChainedSnippets>
+
+## Increasing verbosity with `-v`
+
+You can increase verbosity by passing the `-v` option.
```
Comment (lwronski) [inline website/docs/guides/verbosity.md]:
```suggestion
You can increase verbosity by passing the `-v` option to print debugging logs.
```

File: website/docs/guides/verbosity.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+(...)
+Hello
+
+```
+
+</ChainedSnippets>
+
+## Warnings suppression
+
+Some specific warning logs can be suppressed individually.
```
Comment (lwronski) [inline website/docs/guides/verbosity.md:98]: For now, it should be good enough. However, in the future, we may want to consider adding a table that contains options for suppressing them.

File: website/docs/guides/verbosity.md (hunk 1 above)
Comment (Gedochao) [reply website/docs/guides/verbosity.md]: fixed

File: website/docs/guides/verbosity.md (hunk 2 above)
Comment (Gedochao) [reply website/docs/guides/verbosity.md]: 👍

File: website/docs/guides/verbosity.md (hunk 3 above)
Comment (Gedochao) [reply website/docs/guides/verbosity.md:98]: Agreed.
If we end up having more of them, an automatically generated table would be useful...
We potentially already have a larger one with all the available keys in the `config` sub-command reference doc, but those don't include the corresponding command line options. We'd have to pair them somehow...
But regardless, as you say, it's probably good enough for now.

**Review by lwronski — APPROVED** [review]
LGTM

**Pull request merged**